keyboards/m65/config.h

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2021 Alin M Elena <alinm.elena@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID       0xBABA
+#define MANUFACTURER    m65
+#define PRODUCT         keyboard
```

Comment:
```suggestion
#define MANUFACTURER    Alin Elena
#define PRODUCT         m65
```

Though feel free to change the `MANUFACTURER` value if you don't like having your name used here.

Reply:
thank you all changes done